openedx/features/job_board/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.conf.urls import url
 
-from .views import list_jobs, show_job_detail
+from .views import add_job, list_jobs, show_job_detail
 
 
 urlpatterns = [
-    url(r'^jobs/list/$', list_jobs, name='job_list'),
-    url(r'^jobs/list/[0-9]$', list_jobs, name='job_detail'),
+    url(r'list/$', list_jobs, name='job_list'),
+    url(r'add/$', add_job, name='add_job'),
+    url(r'[0-9]+/$', show_job_detail, name='job_detail'),
```

Comment:
Did we change this url?If so have we made the changes on other parts where it was being used?

Reply:
Good point! If you see in `lms/urls.py` I have moved the prescript in that file where we registered urls of this app, so its just removal of duplication, no change in actual urls or it's usage.

Reply:
Thats right but I see you have removed `/list`as well from job_detail url.But on `lms/urls.py` I see only `/jobs`.

Reply:
Ohk, Yes it was not using anywhere previously. No need to update anywhere else.